vdp/model/v1alpha/model.proto

```diff
@@ -500,6 +502,29 @@ message ClassificationOutputs {
       [ (google.api.field_behavior) = OUTPUT_ONLY ];
 }
 
+message Keypoint {
+  // x coordinate
+  float x = 1 [ (google.api.field_behavior) = OUTPUT_ONLY ];
+  // y coordinate
+  float y = 2 [ (google.api.field_behavior) = OUTPUT_ONLY ];
+  // visibility
+  float v = 3 [ (google.api.field_behavior) = OUTPUT_ONLY ];
+}
+
+message KeypointObject {
+  // Keypoints
+  repeated Keypoint keypoints = 1 [ (google.api.field_behavior) = OUTPUT_ONLY ];
+  // Keypoint score
+  float score = 2 [ (google.api.field_behavior) = OUTPUT_ONLY ];
+}
+
+// KeypointOutput represents the output of keypoint detection task
+// The keypoint detection model do not support batch, so output is singular
+message KeypointOutput {
+  // Keypoint Objects
+  repeated KeypointObject keypoint_output = 1 [ (google.api.field_behavior) = OUTPUT_ONLY ];
```

Comment:
Should we change `keypoint_output` to `keypoint_objects`?

Reply:
@xiaofei-du we could but I just keep consistency with classification/detection
repeated DetectionOutput detection_outputs = 1
repeated ClassificationOutput classification_outputs = 1

Reply:
btw. the keypoint detection model do not support batch so I put output as singular

Reply:
The API is for general cases. Naming convention-wise, it should be `keypoint_outputs`.

Reply:
ok, got it.